webapps/docs/windows-auth-howto.xml

```diff
@@ -101,7 +101,7 @@ policy had to be relaxed. This is not recommended for production environments.
   &lt;service class&gt;/&lt;host&gt;:&lt;port&gt;/&lt;service name&gt;</code>.
   The SPN used in this how-to is <code>HTTP/win-tc01.dev.local</code>. To
   map the user to the SPN, run the following:
-  <source>setspn -A HTTP/win-tc01.dev.local tc01</source>
+  <source>setspn -An HTTP/win-tc01.dev.local tc01</source>
   </li>
```

Comment:
This change is incorrect. This is a command line option hence it needs to be `.. -A HTTP...`

Reply:
> I'm about 2/3 of the way through the commits. With the benefit of experience, 1 PR vs multiple PRs doesn't make that much of a difference. What would have helped was re-basing the PR against master. I've done this locally and there were only a couple of conflicts and it makes it much easier to review.
> 
> I spotted a few unused i18n strings that weren't removed from the translations. That isn't a big deal as the process we use to import/export to/from POEditor will clean those up.
> 
> I have spotted a couple of minor issues. I'll fix those before merging the commits. I'll also comment on the commit in this PR where it makes sense to do so.

lol ! It is so hard to review